coding-intro/coding-intro-core/coding-intro-conditions/comparison-operators.md

```diff
@@ -83,7 +84,7 @@ IF (hungry == true) THEN
 
 ## Revision
 
-The boolean type can only have `true` and ??? as values.
+The boolean type can only have true and ??? as values.
```

Comment:
We need to make type in the gap have codeblocks

How about this:

The boolean type can only have 2 values:
```plain-text
true & ???
```

- false
- negative